library/ssl_tls.c

```diff
@@ -4426,6 +4437,11 @@ int mbedtls_ssl_setup( mbedtls_ssl_context *ssl,
 {
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+    if(check_version_config(conf))
```

Comment:
Can you add a short comment on what we're checking here and how it is going to change in the future?